### ISSUE TITLE
Where are Shopify filters that are not included in this implementation?

### ISSUE THREAD

**Issue by wturnerharris** [Shopify/liquid#1175]
Hello,

There are several Shopify filters that are seemingly not included in this repository.

Does the Shopify team manage those filters separately? Is there a process by which they will eventually be added to this repo? How do [these filters](https://help.shopify.com/en/themes/liquid/filters) make their way to the "spec" as stipulated by this repository?

One example is the [`handleize`](https://help.shopify.com/en/themes/liquid/filters/string-filters#handle-handleize) filter, which I don't see referenced in this repository.

**Comment by shopmike** [comment]
Liquid filters are generally functionality that makes sense to all users of Liquid. It also depends a lot on how these Filters work. Something that uses a third-party library or is possibly built on top of Rails support will not be in Liquid as it is built to be pure ruby and easy to implement.

Shopify filters are part of the core Shopify codebase and as such are not available. The actual docs for Liquid are https://shopify.github.io/liquid/ and the Shopify docs expand on that further with their implemented filters and tags.

**Comment by wturnerharris** [comment]
@shopmike - thanks for the reply. I understand the separation of concerns here, that Shopify Core (proper) adds their own filters and tags for the platform (not public repo), that these are not included in this repo.

I guess my main question is: how do certain filters and tags make it into the main liquid engine?

`handleize` _could_ be used by any user of liquid, so it stands to reason that this _would_ make it into the repository one day, but who makes that decision? Is a feature request to this repo enough? How would it be determined whether a tag or filter is appropriate to be added to the repository? Is it based on upvotes or a body of people?

Basically, we don't want to waste our time with a PR if it's going to be shot down.

**Comment by shopmike** [comment]
Your right `handleize` could be used by any user, but Shopify's implementation isn't suited to being in the core library as it relies on internal objects, etc to provide the best experience.

Things that make it into the core library are usually through multiple requests over time and then eventually it's clear it belongs here. The process of actually making it in starts with a PR. I'm actually proposing one right now https://github.com/Shopify/liquid/pull/1146

However just because a filter doesn't make it into core doesn't mean that it's not useful. Filters are made to be easily installed. So even if it sits in its own repository others will still benefit from the work you have done, especially users of Jekyll and other libraries.

**Comment by wturnerharris** [comment]
@shopmike - thanks for the guidance here. 

Closing this out and will move to PRs, searching open issues and PRs to see if they exist first.

Thanks, again!
